apps/web/src/utils/parse.ts

```diff
@@ -17,12 +17,13 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-export function tryParse(val) {
-  if (val === "undefined" || val === "null") return;
-
+export function tryParse(val: string | null | undefined) {
+  if (!val) {
```

Comment:
Notice how we were comparing with strings (`"undefined"` & `"null"` instead of primitive types `undefined` & `null`) so keep those comparisons. This function is used to turn a string into a JSON object.

Secondly, the `val` parameter must not be `falsy`.

Reply:
Ok